third_party/move/move-vm/runtime/src/session.rs

```diff
@@ -288,6 +288,21 @@ impl<'r, 'l, S: MoveResolver> Session<'r, 'l, S> {
         Ok(instantiation)
     }
 
+    /// Load a module and a function into cache. To load types for the function into
+    /// cache as well, use `load_function`.
+    pub fn load_function_without_types(
```

Comment:
Why do we need it?

Reply:
I need it for use in this PR: https://github.com/aptos-labs/aptos-core/pull/7264.